scripts/pkg/install.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+set -e
+
+reset="\033[0m"
+red="\033[31m"
+green="\033[32m"
+yellow="\033[33m"
+cyan="\033[36m"
+white="\033[37m"
+
```

Comment:
More a DX thing, but we might want to display a message a la

> "You're about to install Serverless to `~/.serverless/bin`"

the user has to confirm so that she knows what's going on. Basically some information about what will happen to the users system.

Reply:
Currently we have `Installing Serverless!`. Do you suggest to replace this message with above one?

Reply:
What I basically mean is a prompt asking the user if he wants us to put some files in his the `~/.serverless/bin` directory. So the script will hang until the user confirms that this is fine.

I saw that with other installers and it gave me a sense of "being in the known" what happens when installing the software. Otherwise it's just an opaque operation and nobody knows in which directories files will be added (that's why I'm for example currently use Nix as a package manager on Mac OS. I hate how other tools just litter my machine).

Reply:
> What I basically mean is a prompt asking the user if he wants us to put some files in his the ~/.serverless/bin directory. 

Is any binary installer, which works that way (via bash script) doing that?

In my understanding user by running install command already states "I want to have this program installed".

I followed practices that are used by other installers I've used (e.g. yarn or nvm), and they do not double confirm with user whether installation should happen

Reply:
Yes, I've seen that a couple of times in the past.

However we can leave it as is for now to not overcomplicate things and see what the community says.